<launch>
    <node pkg="imav" type="cooperation_master" name="cooperation_master" output="screen">
        <param name="exp_high" type="double" value="2.0" />
        <param name="cooperation_master_speed" type="double" value="0.5" />
        <param name="land_dis" type="double" value="50.0" />
    </node>
</launch>
